src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6237,14 +6237,14 @@ internal override IOperation VisitNoneOperation(IOperation operation, int? captu
         private IOperation VisitNoneOperationStatement(IOperation operation)
         {
             Debug.Assert(_currentStatement == operation);
-            VisitStatements(operation.Children.ToImmutableArray());
+            VisitStatements(((Operation)operation).ChildOperations.ToImmutableArray());
```

Comment:
❔ Is there any value in removing these calls to `ToImmutableArray`?

Reply:
Well, it removes at least one boxing, and in some cases an array allocation. More generally, I removed all calls to `Children` in the compiler product layer (not the test layers).

Reply:
I mean is it possible/valuable to try and change this to:

```suggestion
            VisitStatements(((Operation)operation).ChildOperations);
```

Reply:
I don't believe so. This helper is used for a number of cases, and not all of them are visiting all children of a node.